src/components/toast/toast.js

```diff
@@ -450,15 +451,41 @@ function MdToastProvider($$interimElementProvider) {
             }
           }
 
-          // We have to return the innerHTMl, because we do not want to have the `md-template`
+          // We have to return the innerHTML, because we do not want to have the `md-template`
           // element to be the root element of our interimElement.
           return templateRoot.innerHTML;
+        } else if (template) {
+          // We can't sanitize the contents of these templates as it would affect our simple
+          // templates as well as custom templates. It would remove attributes like
+          // aria-live and aria-relevant. It would also remove md-buttons.
+          // TODO Should we go in and call trustAsHtml on those templates farther up the chain?
+          return template;
```

Comment:
@mmalerba there's another edge case here where a custom template may specify `autoWrap = false`. I tested and verified that a `<script>` could be injected here (but I wasn't able to get it to execute). 

I just wanted to confirm that you thought it was worth while to go and try to trust the templates passed in here from `$mdToast.simple()` and `$mdToast.showSimple()`.

Reply:
Browser will (hopefully) stop injected stuff from being executed, but adding stuff like `<img src="/" onerror = "alert(1);"/>` will inject and execute